content/en/idempotence.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Idempotence
+status: Feedback Appreciated
+category: Property
+---
+
+In maths or computer science, idempotence describes an operation 
+that always leads to the same outcome, no matter how many times you execute it. If the parameters are the same, an idempotent operation won't affect the application it calls. As a language-agnostic property, it applies to any programming context.
```

Comment:
The "language-agnostic" comment sounds weird. If it's a property, why would it be language-specific unless it is the property of a programming language? But we aren't speaking about a language here.

Reply:
I agree with @CathPag please delete the last line and mark as completed. I believe this term is ready to be merged.

If you have a chance please link to another authoritative definition of Idempotence, or idempotent, as there are a few solid ones out there.

Thanks!

Reply:
That means just by deleting the last line, it will be okay? I will do it immediately.